rb/lib/selenium/webdriver/firefox/profile.rb

```diff
@@ -24,6 +24,7 @@ class Profile
         include ProfileHelper
 
         VALID_PREFERENCE_TYPES = [TrueClass, FalseClass, Integer, Float, String].freeze
+        WEBDRIVER_PREFS = []
```

Comment:
Do we need this constant? I don't see where it's used.

Reply:
So I was trying to solve this issue because this contact appears here, and throws me the following validation error:

<img width="1558" alt="Screenshot 2024-03-13 at 22 22 03" src="https://github.com/SeleniumHQ/selenium/assets/33221555/d03a19e6-a512-49fb-a58b-3e59536ee8c6">

Reply:
This is a great catch, looks like the constant was lost somewhere during refactorings, but looked like this https://github.com/SeleniumHQ/selenium/blob/560cb6b427780542d7dcd722ab6fc09ff21e65b3/rb/lib/selenium/webdriver/firefox/profile.rb#L9-L15. Let's bring it back at least with `port` and `log_file`.

Reply:
I added log_file & port back